calculator.js

```diff
@@ -1,38 +1,137 @@
-//
-//     // get the user inputs
-// var value1 = prompt('enter first value');
-// var value2 = prompt('enter the second value');
-// var choice = prompt('Choose your action (a)dd (s)ubtract (m)ultiply (d)ivide (mod)ulus') || "a";
-// // console.log(parseInt(value1) + parseInt(value2));
-// switch (choice) {
-//   case "a":
-//     console.log(parseInt(value1) + parseInt(value2));
-//     break;
-//   case "s":
-//     console.log(parseInt(value1) - parseInt(value2));
-//     break;
-//   case "m":
-//     console.log(parseInt(value1) * parseInt(value2));
-//     break;
-//     case "d":
-//       console.log(parseInt(value1)/parseInt(value2));
-//       break;
-//   default:
-//   console.log(parseInt(value1)%parseInt(value2));
-//
-// }
-// // Advanced maths
-// console.log(Math.pow(parseInt(value1), parseInt(value2)));
-// console.log(Math.sqrt(parseInt(value1)));
-
-// //Calculate your BMI in Metric system.
-// var Weight_In_Kg = prompt("Enter your weight in Kilograms");
-// var Hieght_In_M = prompt("Enter your hieght in meters");
-//
-// console.log((parseFloat(Weight_In_Kg))/(Math.pow(parseFloat(Hieght_In_M), 2)));
-//
-// // Calculate your BMI in imperial system.
-// var Weight_In_lbs = prompt("Enter your weight in pounds");
-// var Hieght_In_in = prompt("Enter your hieght in inches");
-//
-// console.log(((parseFloat(Weight_In_lbs))/(Math.pow(parseFloat(Hieght_In_in), 2)))*703);
+
+    // get the user inputs
+var i= 0;
+while (i < 10) {
+var tool = prompt("Which Calculator which you like to use? 'Basic', 'Advanced' , 'BMI' or 'Trip' Calculator ")
```

Comment:
These options are a little too exact - it requires to user to type things like 'Basic' exactly. This means if they typed 'basic', with a lower-case b, for example, it wouldn't work and this would confuse them.

Try to stick to either very simple options e.g. 1. or a. or transform the input. For example, the user could type 'basic', but if you transform that input into all uppercase and check if they've typed 'BASIC', it would be easier.